docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -31,32 +30,80 @@ Some use cases for the Shipping Provider API are:
 
+This article is a guide to developing an app that will make your shipping rates available to merchants and shoppers on demand.
+
 ### Prerequisites
 
-* Required [OAuth](/api-docs/getting-started/authentication#authentication_oauth-scopes) scopes: `Information and Settings`
+* Required [OAuth](https://developer.bigcommerce.com/api-docs/getting-started/authentication/rest-api-authentication#oauth-scopes) scopes: `Information and Settings`
 * BigCommerce [single-click app](https://developer.bigcommerce.com/api-docs/apps/guide/types#single-click)
 
-## Sign-up
+## Shipping app overview
 
-When creating your app, add the shipping data to our Carrier Registry, so it returns to shoppers on the front-end.
+<!--
+    title: ### Carrier Service API
 
-To get your app setup, send an email to
-<a href="mailto:shippingproviderapi@bigcommerce.com">shippingproviderapi@bigcommerce.com</a>.
+    data: //s3.amazonaws.com/user-content.stoplight.io/6012/1552664566834
+-->
 
-Please include:
+### Carrier Service API
 
-- Name
-- Partner ID
-- App ID (see below)
-- Email
-- Description of the app
-- [Your service URLs](#your-service-urls) 
+![### Carrier Service API
+](//s3.amazonaws.com/user-content.stoplight.io/6012/1552664566834 "### Carrier Service API
+")
+
+### Single-carrier vs multi-carrier apps
+
+Only one registered shipping carrier can be associated with an app. This [registered carrier](#definitions) can provide quotes from multiple downstream carriers. 
+
+BigCommerce makes a distinction between single-carrier and multi-carrier shipping providers. The primary difference that this distinction makes is in how the quote displays in the customer's cart at checkout. If your app is registered as a single carrier, the name of the carrier providing the quote will appear beside the name of the shipping quote in the customer's list of shipping rate options. The carrier name will not appear in quotes from multi-carrier apps. The following images illustrate the difference:
+
+<!--
+    title: #### Single-Carrier Quote Example
+
+    data: //raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/shipping-provider-single-carrier-ui.png
+-->
 
+#### Single-Carrier Quote Example
 
-To get your app ID, create an app in [Developer Tools](https://devtools.bigcommerce.com/), and fill out the information on [Step 3 Technical](https://developer.bigcommerce.com/api-docs/apps/guide/publishing#add-technical-information). In the URL, the app will have a unique ID. Send the unique ID in exchange for a carrier ID to test the app.
+![#### Single-Carrier Quote Example](//raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/shipping-provider-single-carrier-ui.png "#### Single-Carrier Quote Example
+")
+
+<!--
+    title: #### Multi-Carrier Quote Example
+
+    data: //raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/shipping-provider-multi-carrier-ui.png
+-->
+
+#### Multi-Carrier Quote Example

```

Comment:
```suggestion
#### Multi-carrier quote example

```
If these are to be headings, use sentence case.

Reply:
done.  pending push.